templates/base.html

```diff
@@ -86,7 +86,7 @@
     <link rel="author" href="/humans.txt">
 
     <link rel="alternate" type="application/rss+xml" title="Python Enhancement Proposals"
-          href="https://www.python.org/dev/peps/peps.rss/">
+          href="https://peps.python.org/peps.rss/">
```

Comment:
Thanks! Little suggestion:

```suggestion
          href="https://peps.python.org/peps.rss">
```